src/btree/bt_page.c

```diff
@@ -359,11 +537,19 @@ __inmem_col_var(WT_SESSION_IMPL *session, WT_PAGE *page, uint64_t recno, size_t
             repeats[repeat_off].recno = recno;
             repeats[repeat_off++].rle = rle;
         }
+
+        /* If we find a prepare, we'll have to resolve it. */
```

Comment:
What do you mean by resolve? Mark them as committed or roll them back? Why?

Reply:
That flag is what triggers the call to __wt_page_inmem_prepare. If you have a better suggestion for wording the comment I'm happy to take it... maybe "we'll have to do __wt_page_inmem_prepare"?

Reply:
I would say, if we find a prepare, we need to instantiate it in the update chain.

Reply:
Fixed.